hw/ip/pwm/data/pwm.hjson

```diff
@@ -0,0 +1,253 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "pwm",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  regwidth: "32",
```

Comment:
could you add `available_output_list` for `pwm_o` signal?

Reply:
My understanding is PWM DOES NOT have dedicated output pins, but rather went to the GPIO MUX.  If that is true I don't think you want these burning 9 more pins as these pins should beaccounted for by GPIO module.

Do you agree?

Reply:
If you want to use GPIO MUX, then you need to define the available_output_list so that pinmux (what you assume as GPIO MUX) can mux those signals with others.

Reply:
I think still available_output_list is missing?